django_girders/configuration/_allauth.py

```diff
@@ -0,0 +1,35 @@
+from typing import Type
+
+from ._base import ComposedConfiguration, ConfigMixin
+
+
+class AllAuthMixin(ConfigMixin):
```

Comment:
```suggestion
class AllauthMixin(ConfigMixin):
```

django-allauth doesn't seem to brand their name as separate words.